src/components/icon.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Icon = ({ color, icon, size, translate, scale }) => {
+  const styles = {
+    svg: {
+      display: 'inline-block',
+      verticalAlign: 'middle',
+    },
+    path: {
+      fill: color,
+    },
+  };
+
+  return (
+    <svg
+      style={styles.svg}
+      width={`${size}px`}
+      height={`${size}px`}
+      transform={`translate(${translate}) scale(${scale})`}
+      viewBox="0 0 1024 1024"
```

Comment:
viewport should have a reasonable default and be passed in as well

Reply:
This still needs to be changed. Not sure what a good default is though.